.github/workflows/Benchmark.yml

```diff
@@ -4,6 +4,10 @@ on:
         types: [created, edited]
     workflow_dispatch:
         inputs:
+            repo:
+                description: Fork to benchmark "user/repo", empty denotes current repo
+                type: string
+                default: ""
```

Comment:
How does the empty string implies default to this repo?

Reply:
The checkout action (line 51) has `with.repository`.  When this is empty, it defaults to the current repo.